From: SeongJae Park <sj@kernel.org>
Date: Mon, 12 May 2025 17:49:20 -0700
Subject: [PATCH] samples/damon/prcl: rename to have damon_sample_ prefix

DAMON sample module, prcl has its name 'prcl'.  It could conflict with
future modules, and not very easy to idnetify it by name.  Use a prefix,
"damon_sample_" for the name.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 samples/damon/prcl.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/samples/damon/prcl.c b/samples/damon/prcl.c
index 5597e6a08ab2..447ccba09e8a 100644
--- a/samples/damon/prcl.c
+++ b/samples/damon/prcl.c
@@ -11,6 +11,11 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 
+#ifdef MODULE_PARAM_PREFIX
+#undef MODULE_PARAM_PREFIX
+#endif
+#define MODULE_PARAM_PREFIX "damon_sample_prcl."
+
 static int target_pid __read_mostly;
 module_param(target_pid, int, 0600);
 
-- 
2.39.5

